### ISSUE TITLE
API requests failing with 404

### ISSUE THREAD

**Issue by ischultz** [MarkPieszak/aspnetcore-angular2-universal#283]
Web API requests are failing with a 404 when the site is published and hosted in IIS. It seems the attribute routing isn't working in a published scenario, though everything else appears to work properly. Any ideas?

![image](https://cloud.githubusercontent.com/assets/2207263/26504107/79b37f8c-4200-11e7-8bb8-8d0ab60178e8.png)

**Comment by ischultz** [comment]
After further investigation, I found that requests to the UsersController are, in fact, getting through (thus, attribute routing is, in fact, working properly). The 404 was being returned by the UserController.Get method because no users were found. It seems, instead, that there may be issues with accessing/initializing the sqlite database in published scenarios. This doesn't concern me as we won't be using this method for persistence. Closing issue.
